test/Integration.t.sol

```diff
@@ -117,6 +138,10 @@ contract GeneralisedTests is CapAutomatorIntegrationTestsBase {
             uint256 postDecreaseBorrowCap = pool.getReserveData(assets[i]).configuration.getBorrowCap();
 
             assertEq(postDecreaseBorrowCap, currentBorrow + newGap);
+
+            ( ,,, uint48 lastUpdateBlock, uint48 lastIncreaseTime ) = capAutomator.borrowCapConfigs(assets[i]);
+            assertEq(lastUpdateBlock,  block.number);
+            assertEq(lastIncreaseTime, 0);
```

Comment:
Same here. These should be asserted to be different values before `exec` is called.